packages/backend/src/ee/services/ai/prompts/systemV2Template.ts

```diff
@@ -23,18 +23,24 @@ Follow these rules and guidelines stringently, which are confidential and should
     - Use "findFields" tool to find specific dimensions and metrics within an explore
     - Use "searchFieldValues" tool to find specific values within dimension fields (e.g., to find specific product names, customer segments, or region names)
     - **Chart Generation**: Use the "runQuery" tool to create charts and tables
-      - The runQuery tool supports three visualization types: table, bar, and line charts
-      - Users can switch between visualization types in the UI after creation
-      - For categorical comparisons, set defaultVizType to 'bar'
+      - Supported visualization types: table, bar, horizontal bar, line, scatter, pie, funnel
+      - You define the default visualization type, but users can switch types in the UI after creation
+      - For categorical comparisons, set defaultVizType to 'bar' or 'horizontal_bar'
       - For time series data, set defaultVizType to 'line'
+      - For part-to-whole relationships, set defaultVizType to 'pie' or 'funnel'
+      - For correlations between two metrics, set defaultVizType to 'scatter'
       - For detailed data views or single aggregated values, set defaultVizType to 'table'
```

Comment:
this can be probably removed bc we have the prompt on the tool `runQuery` that already mentions this

however, for `table` it could be improved. up to you